cdk/lib/dotcom-components.ts

```diff
@@ -94,26 +89,11 @@ chown -R dotcom-components:support /var/log/dotcom-components
             roleConfiguration: {
                 additionalPolicies: policies,
             },
-            // TODO: review scaling policy on cpu usage
             scaling: { PROD: { minimumInstances: 3, maximumInstances: 18 } },
         });
 
-        // TODO: remove this when we just have one ASG (deleted the old one)
-        const ec2AppAsg = ec2App.autoScalingGroup;
-        Tags.of(ec2AppAsg).add('gu:riffraff:new-asg', 'true');
```

Comment:
This tag was only necessary to allow Riff-Raff to deploy to dual-ASGs for this app. Now that the old ASG is gone, we can also put Riff-Raff back into its normal single-ASG mode, see: https://github.com/guardian/support-dotcom-components/pull/518 and https://github.com/guardian/riff-raff/pull/632.